core/src/main/java/org/elasticsearch/cluster/ClusterChangedEvent.java

```diff
@@ -152,46 +168,35 @@ public boolean indexMetaDataChanged(IndexMetaData current) {
         return true;
     }
 
-    public boolean blocksChanged() {
-        return state.blocks() != previousState.blocks();
-    }
-
+    /**
+     * Returns <code>true</code> iff the local node is the mater node of the cluster.
+     */
     public boolean localNodeMaster() {
         return state.nodes().localNodeMaster();
     }
 
+    /**
+     * Returns the {@link org.elasticsearch.cluster.node.DiscoveryNodes.Delta} between
+     * the previous cluster state and the new cluster state.
+     */
     public DiscoveryNodes.Delta nodesDelta() {
         return this.nodesDelta;
     }
 
+    /**
+     * Returns <code>true</code> iff nodes have been removed from the cluster
+     * since the last cluster state.
+     */
     public boolean nodesRemoved() {
         return nodesDelta.removed();
     }
 
-    public boolean nodesAdded() {
```

Comment:
Good point, I only saw they weren't used in ES core so removed them to cleanup, but I'll put them back in (or verify plugins don't use them either, but it is probably not worth doing as part of this commit).